frontend/pages/hosts/details/HostDetailsPage/modals/WipeModal/WipeModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react";
+
+import hostAPI from "services/entities/hosts";
+
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+import Checkbox from "components/forms/fields/Checkbox";
+import { NotificationContext } from "context/notification";
+import { AxiosError } from "axios";
+
+const baseClass = "wipe-modal";
+
+interface IWipeModalProps {
+  id: number;
+  hostName: string;
+  onSuccess: () => void;
+  onClose: () => void;
+}
+
+const WipeModal = ({ id, hostName, onSuccess, onClose }: IWipeModalProps) => {
+  const { renderFlash } = useContext(NotificationContext);
+  const [lockChecked, setLockChecked] = React.useState(false);
+  const [isLocking, setIsLocking] = React.useState(false);
+
+  const onLock = async () => {
```

Comment:
Please update the names of the state variables and change handlers to "wipe" instead of "lock"

Reply:
oh nice thanks. missed that one.